apps/svelte.dev/content/docs/cli/20-commands/10-sv-create.md

```diff
@@ -2,7 +2,7 @@
 title: sv create
 ---
 
-`sv create` sets up a new SvelteKit project, with options to [setup additional functionality](sv-add#Official-add-ons).
+`sv create` sets up a new SvelteKit project, with options to [setup additional functionality](sv-add#Official-integrations).
```

Comment:
manually changing this back so that the site builds

```suggestion
`sv create` sets up a new SvelteKit project, with options to [setup additional functionality](sv-add#Official-add-ons).
```